### PR TITLE
[KO] Retranslate Registration

### PR DESCRIPTION
**Pull request by Ge-n-z**
Previous form had a lot of miswritten words. Changed them.

### CODE REVIEW

File: wiki/Registration/ko.md [hunk 1]
```diff
@@ -4,23 +4,23 @@ no_native_review: true
...
 # 회원가입
 
-*경고: 하나이상의 osu!계정을 가지고 있는 것은 규칙 위반입니다! [osu! rules](/wiki/Rules)!* 참조
+*경고: 두 개 이상의 osu! 계정을 가지고 있는 것은 규칙 위반입니다! [osu! rules](/wiki/Rules)!*를 참조해주세요.
 
-1. osu! [설치](/wiki/Installation) 이후 실행하면, osu!는 로그인 할지 회원가입을 할지 물어봅니다.. 당신이 회원가입을 하기 위해 이글을 읽고잇다면, `Create an account`버튼을 클릭하세요.
+1. osu! [설치](/wiki/Installation) 이후 실행하면, osu!는 로그인 할지 회원가입을 할지 물어봅니다. 당신이 회원가입을 하기 위해 이 글을 읽고있다면, `Create an account`버튼을 클릭하세요.
 
-   ![The in-game registration form](img/ingame-registration.jpg "The registration form")
+   ![The in-game registration form](img/ingame-registration.jpg "회원가입 형식")
```
Comment (hikizisa) [inline wiki/Registration/ko.md:11]: 회원가입 양식 ?

**Review by hikizisa — CHANGES_REQUESTED** [review]
overall good but I think one wording could be changed

File: wiki/Registration/ko.md (hunk 1 above)
Comment (Ge-n-z) [reply wiki/Registration/ko.md:11]: 깜빡하고 수정 못한 부분이 있었네요 감사합니다!

Comment (TPGPL) [inline wiki/Registration/ko.md:11]: `The in-game registration form` needs to be translated

Comment (Ge-n-z) [reply wiki/Registration/ko.md:11]: Oops sorry, I thought this problem was solved

**Comment by TicClick** [comment]
@hikizisa @Ge-n-z is this ready for merge?

**Review by TPGPL — APPROVED** [review]

**Pull request merged**